tutorials/migration.md

```diff
@@ -292,134 +237,112 @@ To get a list of all currently installed items run:
-as if this were the first installation and configuration (e.g. you need to
-generate a new pin code for Nest).
+One important thing to note is that bindings and add-ons that require extra steps to register with a cloud service like Nest will require reregistering again as if this were the first installation and configuration (e.g. you need to generate a new pin code for Nest).
 
-Unlike in openHAB 1.x, transformations are not automatically included in
-openHAB 2. Make sure you include these in your installation as well.
+Unlike in openHAB 1.x, transformations are not automatically included in openHAB 2. 
+Make sure you include these in your installation as well.
 
 ### Installing Unofficially Supported openHAB 1.x Add-ons
 
 Skip this section if all the add-ons you need have been installed already.
 
-First check the [list of add-ons that are known not to work in openHAB 2]({{base}}/addons/1xaddons.html#currently-incompatible-1x-add-ons)
-and make sure yours is not among them.
+First check the [list of add-ons that are known not to work in openHAB 2]({{base}}/addons/1xaddons.html#currently-incompatible-1x-add-ons) and make sure yours is not among them.
 
-Next install the openHAB 1.x compatibility layer using the Karaf Console
-instructions above.
+Next install the openHAB 1.x compatibility layer using the Karaf Console instructions above.
 
-Copy your openhab.cfg file to `<openHAB 2 conf>/services`. If you are running an apt-get
-installed openHAB 1.x openhab.cfg is located in `/etc/openhab/configurations`.
-For an apt-get installed openHAB 2 the `<openHAB 2 conf>` folder is located in
-`/etc/openhab2`. For manually installed openHAB 1.x this file is located
-in `<openhab home>/configurations` and in manually installed openHAB 2 it is
-`<openhab home>/conf`.
+Copy your openhab.cfg file to `<openHAB 2 conf>/services`. 
+If you are running an apt-get installed openHAB 1.x openhab.cfg is located in `/etc/openhab/configurations`.
+For an apt-get installed openHAB 2 the `<openHAB 2 conf>` folder is located in `/etc/openhab2`. 
+For manually installed openHAB 1.x this file is located in `<openhab home>/configurations` and in manually installed openHAB 2 it is `<openhab home>/conf`.
 
-This file will have a lot of redundant configuration information in it that you
-have already moved over to individual binding .cfgs. Make sure to comment those
-out and just leave the config parameters for the bindings you are manually
-installing.
+This file will have a lot of redundant configuration information in it that you have already moved over to individual binding .cfgs. 
+Make sure to comment those out and just leave the config parameters for the bindings you are manually installing.
 
-Now copy the add-on's jar files that you want to install to the `<openHAB conf>/addons` folder for
-openHAB 2. For example:
+Now copy the add-on's jar files that you want to install to the `<openHAB conf>/addons` folder for openHAB 2. 
```

Comment:
According to #189 we should use a consistent `$FOLDER` notation for all articles.